plugins/bundle/config.go

```diff
@@ -10,6 +10,8 @@ import (
 	"path"
 	"strings"
 
+	"github.com/pkg/errors"
```

Comment:
Let's not introduce this in new code. fmt.Errorf with %w should do the trick.

Reply:
Updated to use `fmt.Errorf`